src/Compilers/CSharp/Test/Emit2/Semantics/PrimaryConstructorTests.cs

```diff
@@ -3866,9 +3864,6 @@ protected void Handle5(SyntaxNodeAnalysisContext context)
                             case "B..ctor([System.Int32 Y = 1])":
                                 Interlocked.Increment(ref FireCount9);
                                 break;
-                            case "B":
-                                Interlocked.Increment(ref FireCount17);
-                                break;
```

Comment:
> Why remove this case, rather than update the expected number of hits in the tests?

I think removal is preferred. We do not expect any hits and the `default` case verifies that in a reliable way.